### ISSUE TITLE
Accessibility: Missing alt text on Ottawa logo img

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#7]
WCAG 2.0 - Success Criteria - 1.1.1 Non text content (level A) violation
1. Go to data.ottawa.ca
2. Ottawa logo is without alt text

Resolution: Add alt text to logo below
![missin alt text on ottawa logo](https://cloud.githubusercontent.com/assets/5271253/8012950/4282304e-0b93-11e5-8b25-28ef7c94e8f4.JPG)
![html_ line 94_column 138](https://cloud.githubusercontent.com/assets/5271253/8012956/47d3ebf0-0b93-11e5-8cb1-2db226aaccdd.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar I don't know if you are interested or have the bandwidth to also validate fixed issues, so I closed some of them myself. If you do end up doing another run-through and find that some of the closed issues persist, please feel free to re-open them.
